Contract Template/description.ext

```diff
@@ -30,4 +30,8 @@ enableTargetDebug = 1;
 
 // Other settings -  Do Not Edit.
 cba_settings_hasSettingsFile = 1;  // Enables use of "cba_settings.sqf", remove if file is not present (will crash)
-corpseManagerMode = 0; // No bodies will be cleaned up
\ No newline at end of file
+corpseManagerMode = 0; // No bodies will be cleaned up
+disableChannels[] = { // Disables unused chat channels
```

Comment:
Should probably put in its own section near Header or respawn, as I can see this being edited for specials.

Reply:
Agreed, corpse manager can be really handy for something like a large defense mission as bodies can really tend to stack up there. If there's space, could add in the comment what different modes are there (0 = no bodies, 1= xx etc.).

Channel, the global one can be handy in the odd PvP mission (rare, but in case something breaks or whatever there's at least a way to communicate cross-teams about whatever)

Reply:
> Agreed, corpse manager can be really handy for something like a large defense mission as bodies can really tend to stack up there.

Also removes player bodies after respawn (no gear back).

Reply:
I usually go through the corpse manager on defense missions to ensure that the case of losing a player body is extremely unlikely while still cleaning up the AI bodies. (We really don't want people messing with it unless it REALLY needs to be used.)

Shifted `disableChannels` into it's own category a bit further up.